subt/tf_detector.py

```diff
@@ -10,8 +10,8 @@
 PATH_TO_PB_GRAPH = "subt/tf_models/frozen_inference_graph.pb"
 PATH_TO_CV_GRAPH = "subt/tf_models/cv_graph.pbtxt"
 
-ARTF_NAME = np.array(["backpack", "survivor", "phone", "rope", "helmet", "robot"])
-MIN_SCORES = np.array([0.84, 0.3, 0.4, 0.2, 0.45, 1.0])
+ARTF_NAME = np.array(["backpack", "survivor", "phone", "rope", "helmet", "robot", "breadcrumb", "nothing"])
+MIN_SCORES = np.array([0.36, 0.25, 0.1, 0.1, 0.4, 1.0, 1.0, 1.0])
```

Comment:
Why not a threshold for helmet somewhere between 0.45 and 0.5? The lowest scoring positives in your graph are just below 0.5 score.

Reply:
I suppose similar to the other comments: The graphs do not contain all available data.

Reply:
I assume more positive cases near to 0.4 in the real data.